server/schemas/group.graphql

```diff
@@ -0,0 +1,47 @@
+type Group implements Node {
+    id: ID!
+    schemaId: ID!
+    projectId: ID!
+    name: String!
+    description: String!
+    key: String!
```

Comment:
is this key required? is not schema field key enough?

Reply:
yes sure, If you check the Figma the group should have a key field in a similar way to the model
also, the group schema field is not a group.... the schema group field is pointing at the group but they are both different domains

Reply:
>  the group schema field is not a group.... the schema group field is pointing at the group but they are both different entities

this is clear to me, but how we will use the group key?